packages/abstractions/src/serialization/untypedArray.ts

```diff
@@ -0,0 +1,19 @@
+import { isUntypedNode, UntypedNode } from "./untypedNode";
+
+export class UntypedArray extends UntypedNode {
```

Comment:
I'm a bit affraid that javascript/typescript developers will hate us if we make them use classes to build JSON payloads (or other formats).
Do you think we could move to an interface approach instead?

Reply:
I hear you on this. 
Let me spike this one and propose an alternative so that we can look at to evaluate the two.